### PR TITLE
Contactform endpoint v1

### PR DESCRIPTION
**Pull request by mmcgowanBOS**

### CODE REVIEW

File: docroot/modules/custom/bos_components/modules/bos_email/bos_email.module [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
```
Comment (davidrkupton) [inline docroot/modules/custom/bos_components/modules/bos_email/bos_email.module:1]: If you want, you can omit this file if its not used.  We dont have to have a module file.

Comment (mmcgowanBOS) [reply docroot/modules/custom/bos_components/modules/bos_email/bos_email.module:1]: OK. I usually include it by default out of habit.

File: scripts/.config.yml [hunk 2]
```diff
@@ -77,6 +77,20 @@ build:
         # Set to 'true' or 'false'.  False means no configs will be imported during build/deploy
         # Note: if the commit is a hotfix, then configs wont be imported for expediency.
```
Comment (davidrkupton) [inline scripts/.config.yml]: You probably dont want to commit this config file to the develop branch.

**Review by davidrkupton — APPROVED** [review]
Brilliant !!
I have 2 comments about excluding files.  the .module file can stay if you want, but I dont think the changes to the config file should be committed.  Its irrelevant to the travis/acquia build, its just whoever pulls develop later will get those changes and they are not going to be relevant to them.

File: scripts/.config.yml (hunk 2 above)
Comment (mmcgowanBOS) [reply scripts/.config.yml]: I reset it to the default values. I thought it was good practice to keep it in develop as that's the branch devs use to checkout to their local and then deploy when / if neccessary.

**Pull request merged**